enhancements/machine-api/opt-out-machine-health-checking.md

```diff
@@ -0,0 +1,378 @@
+---
+title: opt-out-machine-health-checking
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-05-18
+last-updated: 2020-05-18
+status: implementable
+see-also:
+  - "/enhancements/machine-api/machine-health-checking.md"  
+---
+
+# Opt-Out Machine Health Checking
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
```

Comment:
I'm against doing enabling MHC by default, I prefer to keep it opt-in, similar to what we do with the autoscaler.

Reason being, by opting-in, you are spending the time to learn and understand at least the very basics of the MHC system.

Enabling by default IMO increases the operational knowledge required to run a basic cluster.  In an ideal world, we can set it and forget it, but I would not personally want something on my cluster that is deleting machines without an understanding of how it works and how I adjust it.